activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -533,6 +533,19 @@ def drop_table(table_name, **options)
         execute "DROP TABLE#{' IF EXISTS' if options[:if_exists]} #{quote_table_name(table_name)}"
       end
 
+      # Drops the given tables from the database.
+      #
+      # [<tt>:force</tt>]
+      #   Set to +:cascade+ to drop dependent objects as well.
+      #   Defaults to false.
+      # [<tt>:if_exists</tt>]
+      #   Set to +true+ to only drop the table if it exists.
+      #   Defaults to false.
+      def drop_tables(*table_names, **options)
```

Comment:
Instead of adding a new method with the same implementation as above, we can just change `drop_table` to accept an array.

Reply:
I'm not sure since the drop_table is reversible but drop_tables wouldn't be.

Reply:
Kind of the same scenario on remove_column vs remove_columns

Reply:
But to `drop_table` to be reservible we need a block. We are already doing a check for arguments to revert, so we can just add another check.

Reply:
But then we would raise in the case drop_table receives more than one argument + block?

Reply:
Yes.